packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -133,21 +158,37 @@ function getLoadDocumentsOptions(filename: string) {
   return loadTypedefsConfig
 }
 
-function getPluginConfig() {
+function getPrismaModels() {
   let prismaModels: Record<string, string> = {}
+
+  // Extract the models from the prisma client and use those to
+  // set up internal redirects for the return values in resolvers.
+  const localPrisma = require('@prisma/client')
+  prismaModels = localPrisma.ModelName
+
+  // This isn't really something you'd put in the GraphQL API, so
+  // we can skip the model.
+  if (prismaModels.RW_DataMigration) {
```

Comment:
This used to be wrapped in a try ... catch. Now it isn't, so this blows up if the prisma client isn't generated